app/assets/static/picker.html

```diff
@@ -0,0 +1,42 @@
+<html>
+  <head>
+      <title>Picker Page</title>
+      <link rel="stylesheet" media="screen" href="//unpkg.com/accessible-typeahead@0.4.0/examples/styled.css" />
+  </head>
+  <body>
+    <div class="form-group">
+      <label for="data-picker">Picker Label:</label>
+      <div id="picker" class="typeahead-wrapper form-control-1-2"></div>
```

Comment:
This code is dummy and will get ripped out but generated HTML from the tool so please ignore.